### ISSUE TITLE
Run the Codellama model quickly based on the Hugging Face pretrained model

### ISSUE THREAD

**Issue by yinhong-hu** [meta-llama/codellama#83]
Hello,

I've noticed that for some learners who are just getting started with the Codellama model, running a Codellama model using your code directly can be a bit challenging. To make it easier for everyone to quickly run their own Codellama model, I plan to create a simple script based on the Codellama pretrained model downloaded from Hugging Face. With this script, users can start using Codellama effortlessly. I intend to submit a pull request for this. Please keep an eye out for it.

**Comment by lvhan028** [comment]
Hi, @hyhhyh402 have you submitted a PR?

**Comment by yinhong-hu** [comment]
yes, my PR is in https://github.com/facebookresearch/codellama/pull/85

**Comment by yinhong-hu** [comment]
> Hi, @hyhhyh402 have you submitted a PR?
yes, my is in https://github.com/facebookresearch/codellama/pull/85

**Comment by realhaik** [comment]
> Hello,
> 
> I've noticed that for some learners who are just getting started with the Codellama model, running a Codellama model using your code directly can be a bit challenging. To make it easier for everyone to quickly run their own Codellama model, I plan to create a simple script based on the Codellama pretrained model downloaded from Hugging Face. With this script, users can start using Codellama effortlessly. I intend to submit a pull request for this. Please keep an eye out for it.

On problem, the HF llama models are trash, and the answers are completely broken. It is very sad that people don't understand this.

**Comment by bronzwikgk** [comment]
Where can I find the Url to the script.
